src/track/cue.cpp

```diff
@@ -87,6 +89,53 @@ void Cue::setSource(CueSource source) {
     emit(updated());
 }
 
+Cue::CueStatus Cue::getStatus() const {
+    CueStatus status = CueStatus::DISABLED;
+    switch(getType()) {
+        case CueType::LOOP:
+            if (getPosition() != -1 && getLength() > 0) {
+                status = CueStatus::ENABLED;
```

Comment:
"Enabled" is ambiguous in this case.
How about "valid"

Reply:
ok